apps/judicial-system/backend/src/app/modules/case/case.controller.ts

```diff
@@ -313,9 +313,13 @@ export class CaseController {
       case CaseTransition.REJECT:
       case CaseTransition.DISMISS:
       case CaseTransition.COMPLETE:
-        update.rulingDate = isIndictmentCase(theCase.type)
-          ? nowFactory()
-          : theCase.courtEndTime
+        if (isIndictmentCase(theCase.type)) {
+          update.postponedIndefinitelyExplanation = null
+          update.courtDate = null
```

Comment:
Is it really necessary to clear these variables for a completed case? They are used in the court process which will no longer be accesible and in `InfoCardScheduledIndictment` which should not be shown for completed cases.

Also note that even if we clear `courtDate`, `arraignmentDate` is still set.